mappings/net/minecraft/RegistryProvider.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_5423 net/minecraft/RegistryProvider
```

Comment:
Package is a thing

Reply:
sorry i dont seem to understand, the class is located at net/minecraft

Reply:
liach means that the class name needs to be assigned to a subpackage, right now you just have it in the root minecraft package `net.minecraft.RegistryProvider`

Reply:
ah i think i get it, i thought the subpackages are provided by Mojang and the classes are already in their correct packages